regression-test/suites/correctness/test_backends_table.groovy

```diff
@@ -1,23 +0,0 @@
-// Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
We can modify this to test `backends()` tvf

Reply:
ok